modules/connectors/ninja_van/karrio/providers/ninja_van/shipment/cancel.py

```diff
@@ -0,0 +1,38 @@
+
+import typing
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.ninja_van.error as error
+import karrio.providers.ninja_van.utils as provider_utils
+import karrio.providers.ninja_van.units as provider_units
+
+
+def parse_shipment_cancel_response(
+    _response: lib.Deserializable[dict],
+    settings: provider_utils.Settings,
+) -> typing.Tuple[models.ConfirmationDetails, typing.List[models.Message]]:
+    response = _response.deserialize()
+    messages = error.parse_error_response(response, settings)
+    success = True  # compute shipment cancel success state
```

Comment:
This also needs to be implemented. Since the response is 
```
{
  "trackingId": "string",
  "status": "string",
  "updatedAt": "string"
}
```

based on the NinjaVan Cancel Order API https://api-docs.ninjavan.co/en#tag/Order-API/paths/~1%7BcountryCode%7D~12.2~1orders~1%7BtrackingNo%7D/delete

You can do something like 

```
# If the status is OK by example
success = response.get("status") == "OK"